Formula/include-what-you-use.rb

```diff
@@ -57,8 +57,8 @@ def install
     # formula. This would be indicated by include-what-you-use failing to
     # locate stddef.h and/or stdlib.h when running the test block below.
     # https://clang.llvm.org/docs/LibTooling.html#libtooling-builtin-includes
-    (libexec/"lib").install_symlink llvm.lib/"clang"
-    (libexec/"include").install_symlink llvm.include/"c++"
+    (libexec/"lib").install_symlink llvm.opt_lib/"clang"
+    (libexec/"include").install_symlink llvm.opt_include/"c++"
```

Comment:
This doesn't help -- `llvm.lib == llvm.opt_lib` outside the `llvm` formula.

https://rubydoc.brew.sh/Formula.html#prefix-instance_method

Reply:
Ah yes.

I'll undo, but it will be interesting to see if a revision bump helps here at least.

Reply:
Answer: it does. Not entirely sure why.

Reply:
Seems like we were symlinking to the Cellar? `c++ -> ../../../../llvm@11/11.1.0/include/c++`

Reply:
Yea, not surprising. Revision bumps always help IWYU. Though I have to say I'm confused as to why it would now need one...

Reply:
Because if we are symlinking to the Cellar then the path is now `11.1.0_1` rather than `11.1.0`.

Reply:
Ah, yes. We were symlinking to the cellar. I do remember noting that `install_symlink` seems a bit eager about resolving symlinks that you pass to it.

Reply:
https://github.com/Homebrew/brew/pull/11351